formidling/src/main/java/no/nav/ung/sak/formidling/vedtak/DetaljertResultatUtlederImpl.java

```diff
@@ -206,4 +213,12 @@ private static <V> boolean innholderBare(Set<V> set, V... value) {
         return set.equals(Arrays.stream(value).collect(Collectors.toSet()));
     }
 
+    private static boolean innholderBareOgOptionallyUttalelse(Set<BehandlingÅrsakType> set, BehandlingÅrsakType behandlingÅrsakType) {
+        return set.contains(behandlingÅrsakType) && (set.size() == 1 || set.size() == 2 && set.contains(BehandlingÅrsakType.UTTALELSE_FRA_BRUKER));
+    }
+
+    private static boolean harAvslåttVilkår(Set<DetaljertVilkårResultat> avslåtteVilkår, VilkårType vilkårType) {
+        return avslåtteVilkår.stream().anyMatch
+            (it -> it.vilkårType().equals(vilkårType));
```

Comment:
Du kan bruke `==` istedenfor `equals` da du sammenlikner enums :)

Reply:
Eg liker egentlig å bruke equals for enumer også, for då er man ikkje avhengig av å vite at det er ein enum for å lese koden riktig.